src/components/Banner/Banner.tsx

```diff
@@ -31,40 +41,82 @@ export default function Banner({ queryRef, text, requireAuth = false }: Props) {
 
   const isAuthenticated = Boolean(query.viewer?.user?.id);
 
-  const [dismissed, setDismissed] = usePersistedState(
-    JAN_18_MIGRATION_BANNER_DISMISSED_LOCAL_STORAGE_KEY,
-    false
-  );
+  const [dismissed, setDismissed] = usePersistedState(LOCAL_STORAGE_KEY || '', false);
```

Comment:
do we need to reassign to the const on L26? why not just keep this as `localStorageKey`